dashboard/public/frequency/frequency.html

```diff
@@ -4,6 +4,8 @@
 
 <script type="text/javascript" src="/frequency/frequency.js"></script>
 
+<div id="bubble"></div>
```

Comment:
Hmm. Because both of these apps are build around the idea of working with some amount of input text, I decided for both of them to put the non-markdown instructions in as the "default" text. I suppose there is the open question of how valuable it is to provide them with something they can click on to show the markdown instructions again.

@sfilman?